### ISSUE TITLE
Android app with HTTP connection configured reports nothing

### ISSUE THREAD

**Issue by spinscale** [owntracks/android#522]
* **App build number**: 1.2.7 (12701)
* **Android version**: 6.0.1
* **Device**: Motorola G3

I have installed owntracks, but it is not reporting anything, even when clicked in the UI to report (the location in the map widget is reported correctly, I can see where I am). I am pretty sure, that it is owntracks not doing anything. I can call the HTTP endpoint with a REST client from that phone and it works. I also tried connecting to a system in the local network in owntracks, which does not work either, so internet connection issues do not seem to be a thing.

I also know, that this used to work a couple of months ago on this phone, so something might have changed with an upgrade.

If there is anything I can do to help debugging the issue, I am happy to help (just not an android expert, just a user).

**Comment by binarybucks** [comment]
A couple of things to check:
- do the logs of your backed indicate any connection at all? 
- what is shown on the status page of the app?
- have you granted the location permission? Without, owntracks cannot send anything.

**Comment by spinscale** [comment]
I removed and reinstalled everything from scratch. And it works now. One difference is, that before the 'Message queued' popup, when clicking on the report button did not show up, now it does. I cant tell what the cause of this was.

I had granted the permissions. Could they have been revoked by any system upgrade?

The endpoint state message now is correctly `Response 200` and did not show anything before.

Sorry for any inconvenience, this is all I can come up with to look up. I am gonna close this, unless you know where else I could have looked.
